k8s/templates/studio-deployment.yaml

```diff
@@ -50,9 +74,6 @@ spec:
           periodSeconds: 2
           # fail 3 times before taking this app off the routing table
           failureThreshold: 3
-        volumeMounts:
-        - mountPath: /app/contentworkshop_static/
-          name: staticfiles
```

Comment:
we move the volume mount from the `app` container to the `collecstatic` initContainer.